src/service/viewer-impl.js

```diff
@@ -775,7 +775,8 @@ export class Viewer {
    * @return {!Promise}
    */
   requestFullOverlay() {
-    return this.sendMessageUnreliable_('requestFullOverlay', {}, true);
+    return /** @type {!Promise} */ (
```

Comment:
Because CC is bad at templates and this message can return null. Unfortunately the assert cannot produce the right type here.